api/v1/routes/activity_log.py

```diff
@@ -0,0 +1,75 @@
+from typing import List
+from fastapi import APIRouter, HTTPException, Depends, status
+from api.utils.dependencies import get_current_user, get_super_admin
+from sqlalchemy.orm import Session
+from api.v1.services.activity_log import ActivityLogService
+from api.v1.models import User
+from api.db.database import get_db
+from api.v1.schemas.activity_log import ActivityLogCreate, ActivityLogResponse, GetActivityLogResponse
+from datetime import datetime
+
+"""
+Router module for activity-logs
+POST, GET
+"""
+
+router = APIRouter(tags=["Activity-logs"])
+
+@router.post("/activity-logs/create", response_model=ActivityLogResponse)
```

Comment:
no need for the create verb here

```suggestion
@router.post("/activity-logs/", response_model=ActivityLogResponse)
```